modules/structs/actions_visibility.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package structs
+
+// ActionsVisibility defines the activities shown
+type ActionsVisibility int
```

Comment:
A naming problem: in Gitea, "actions" has a special meaning now: the action runner and related features.

So I would prefer to use the old name "activity" to distinguish these concepts.

Actually when I read this PR's title, the first thought is that "Oh, it is related to Gitea Actions", but it is not .....